php/ad-servers/class-ad-layers-dfp.php

```diff
@@ -493,7 +493,7 @@ private function ad_unit_js( $ad_layer ) {
 					$custom_targeting = null;
 					if ( ! empty( $this->ad_units[ $unit_key ] ) ) {
 						$custom_targeting = $this->ad_units[ $unit_key ];
-					} elseif ( empty( $this->ad_units[ $unit_key ] ) && ! empty( $ad_unit['custom_targeting'] ) ) {
+					} elseif ( ! empty( $ad_unit['custom_targeting'] ) ) {
```

Comment:
I'm concerned about the change in logic here. Can you explain how this doesn't break anything?